### PR TITLE
should closing flag be set false in HRegion.java

### PR DESCRIPTION
**Pull request by zhengxle**
refer: HBASE-21593

### CODE REVIEW

**Comment by HorizonNet** [comment]
@zhengxle Could you please update the commit message to include the Jira ticket ID? See the appropriate [documentation](https://hbase.apache.org/book.html#_commit_message_format) for more details.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 238 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 256 | master passed |
| +1 | compile | 52 | master passed |
| +1 | checkstyle | 71 | master passed |
| +1 | shadedjars | 257 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 148 | master passed |
| +1 | javadoc | 33 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 257 | the patch passed |
| +1 | compile | 55 | the patch passed |
| +1 | javac | 55 | the patch passed |
| +1 | checkstyle | 72 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 269 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 502 | Patch does not cause any errors with Hadoop 2.7.4 or 3.0.0. |
| +1 | findbugs | 145 | the patch passed |
| +1 | javadoc | 31 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 15641 | hbase-server in the patch failed. |
| +1 | asflicense | 26 | The patch does not generate ASF License warnings. |
| | | 18118 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.replication.multiwal.TestReplicationSyncUpToolWithMultipleWAL |
|   | hadoop.hbase.client.TestSnapshotTemporaryDirectoryWithRegionReplicas |
|   | hadoop.hbase.master.TestAssignmentManagerMetrics |
|   | hadoop.hbase.client.TestAdmin1 |
|   | hadoop.hbase.client.TestFromClientSideWithCoprocessor |
|   | hadoop.hbase.client.TestFromClientSide |
|   | hadoop.hbase.master.TestSplitWALManager |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-105/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/105 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux babe299e61b0 4.4.0-137-generic #163-Ubuntu SMP Mon Sep 24 13:14:43 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 11547b79f3 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-105/1/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-105/1/testReport/ |
| Max. process+thread count | 4850 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-105/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 47 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 1 | The patch does not contain any @author tags. |
| -0 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 270 | master passed |
| +1 | compile | 56 | master passed |
| +1 | checkstyle | 80 | master passed |
| +1 | shadedjars | 282 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 252 | master passed |
| +1 | javadoc | 36 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 261 | the patch passed |
| +1 | compile | 54 | the patch passed |
| +1 | javac | 54 | the patch passed |
| +1 | checkstyle | 77 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 282 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 548 | Patch does not cause any errors with Hadoop 2.7.4 or 3.0.0. |
| +1 | findbugs | 250 | the patch passed |
| +1 | javadoc | 34 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 12146 | hbase-server in the patch passed. |
| +1 | asflicense | 27 | The patch does not generate ASF License warnings. |
| | | 14768 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-105/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/105 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 790757bd8737 4.4.0-144-generic #170~14.04.1-Ubuntu SMP Mon Mar 18 15:02:05 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / f30d6c958a |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-105/2/testReport/ |
| Max. process+thread count | 4181 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-105/2/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java [hunk 1]
```diff
@@ -1710,6 +1710,7 @@ public void setTimeoutForWriteLock(long timeoutForWriteLock) {
       LOG.info("Closed " + this);
       return result;
     } finally {
+      this.closing.set(false);
```
Comment (saintstack) [inline hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java]: Not sure how this addresses problem in the issue and what the benefit is here?

**Comment by saintstack** [comment]
Closed old PR that has gone months w/o update.

**Pull request closed without merging**